types/banana-i18n.d.ts

```diff
@@ -0,0 +1,29 @@
+export interface Messages {
+	[ messageKey: string ]: string; 
+}
```

Comment:
This doesn't take into account the `@metadata` property which is not a string

Reply:
Thanks for telling me this. However could you please point to file where the tag is used. This PR is open since a year and meanwhile things probably changed and I am engaged in some other projects, which do not relate to Wikipedia tooling. Also I implemented a lib which serves the same purpose in Kotlin and I cannot recall from there any tag of this kind.

Reply:
See the documentation. The example *.json files have the @metadata.

Reply:
As to how this can be solved, i tried 
```
export interface Messages {
	'@metadata': any;
	[ messageKey: string ]: string; 
}
```

in my fork but this doesn't actually work. TS still complains of `@metadata` not matching the index signature :(

Reply:
Thx, for pointing on this. I will look into it soonish (I have currently some deadlines) and try some TS magic.
By the way...if I am correct I still have the TS definitions for the Vue module of Banana, which are based on this PR (iff I recall that correctly). Are you also interested in those?

Reply:
Okay...After a quick look...I know, why your forks will not work...I'll explain it, when I done the changes.

Reply:
I changed it, so that now `@metadata` is accepted by the compiler. However it technically allows now objects also for other keys. This is due to the nature of index structure in typescript, which cannot differ between specific keys, since only strings and number are allowed. But is much better then untyped or `any`.